README.rst

```diff
@@ -157,25 +157,31 @@ follows:
 
 .. code-block:: console
 
-   $ less environment/Dockerfile
-   # Start from the ROOT6 base image:
-   FROM reanahub/reana-env-root6
-
-   # Install HFtools:
-   RUN apt-get -y update && \
-       apt-get -y install \
-          python-pip \
-          zip && \
-       apt-get autoremove -y && \
-       apt-get clean -y
-   RUN pip install hftools
-
-   # Mount our code:
-   ADD code /code
-   WORKDIR /code
-
-We can build our analysis environment image and give it a name
-``reanahub/reana-demo-bsm-search``:
+    # Start from the ROOT6 base image:
+    FROM reanahub/reana-env-root6
+
+    # Install HFtools:
+
+    # Update and install dependendies
+    RUN apt-get -y update && \
+        apt-get -y install \
+           python-pip \
+           zip && \
+        apt-get autoremove -y && \
+        apt-get clean -y
+
+    RUN pip install hftools==0.0.1
+
+    # Mount our code:
+    ADD code /code
+    WORKDIR /code
+
+    # Mount our code:
+    RUN adduser atlas && usermod -aG root atlas
+    USER atlas
```

Comment:
I had a look and adding default non-root user should not be necessary... The trouble is Python clashing versions, e.g. `hftools` is not Python-3 ready yet (so we need to use Python 2 with  recent ROOT) and the old ROOT image seem to have been built for other Python.... I'm on solving this, but it looks like we won't need to introduce user in any case, so I'll probably make a new PR

Reply:
Superseded by #17